tests/RobotFramework/README.md

```diff
@@ -110,6 +110,22 @@ Checkout the [dev container instructions](../../docs/src/developer/DEV_CONTAINER
 
 3. Follow the console instructions and edit the `.env` file which was created by the `./bin/setup.sh` script
 
+    It is recommended to provide a local certificate authority (CA) signing certificate so that the system tests can utilize it to generate the device certificates. To use your own signing certificate you need to add the following keys which contain the base64 encoded contents of the CA key and public certificate.
+
+    For example, assuming you have the `~/tedge-ca.key` (private key) and the `~/tedge-ca.crt` public certificate, you can run the following on the console:
```

Comment:
I would add the command to create such a signing certificate.

```
openssl req \
    -new \
    -x509 \
    -days 999 \
    -extensions v3_ca \
    -nodes \
    -subj "/O=thin-edge/CN=tedge-ca" \
    -keyout ~/tedge-ca.key \
    -out ~/tedge-ca.crt
```

Reply:
Yeah I was expecting this comment :D

I initially avoided it because we already partially describe it in the online docs, [generating-certificates](https://thin-edge.github.io/thin-edge.io/next/operate/security/mqtt_tls_configuration/#generating-certificates)

But I'll add the oneliner to the docs.

Reply:
I've added the one-liner to generate a CA certificate

Resolved by bb43d2f8d13cf481d2a89c029869a35a089deb25